kits/find-your-hotel/.env.example

```
@@ -1,5 +1,5 @@
-LAMATIC_PROJECT_ENDPOINT= Your Lamatic GraphQL endpoint
-LAMATIC_FLOW_ID= Hotel finder flow ID
-LAMATIC_AGENT_ID= Agent ID  
-LAMATIC_PROJECT_ID= Project ID from Lamatic Studio
+LAMATIC_PROJECT_ENDPOINT= Your Lamatic GraphQL endpoint
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)